$NetBSD: patch-aa,v 1.10 2007/06/12 03:02:34 dmcmahill Exp $

Use a pointer, instead of an array, to avoid an error on Darwin.
Patch from DarwinPorts.

--- libguile/guile.c.orig	2006-02-18 06:08:45.000000000 +0000
+++ libguile/guile.c
@@ -68,7 +68,7 @@ main (int argc, char **argv)
 {
 #if !defined (__MINGW32__)
   /* libtool automagically inserts this variable into your executable... */
-  extern const lt_dlsymlist lt_preloaded_symbols[];
+  extern const lt_dlsymlist *lt_preloaded_symbols;
   lt_dlpreload_default (lt_preloaded_symbols);
 #endif
   scm_boot_guile (argc, argv, inner_main, 0);
